config/rbac/role.yaml

```diff
@@ -59,3 +59,14 @@ rules:
   - get
   - list
   - patch
+- apiGroups:
+  - sriovnetwork.openshift.io
```

Comment:
I'm not sure if the group is right, openshift.io seems rather odd

Reply:
https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/578098b65353bc9ce40103e1ee3b660a22e0ae13/deployment/sriov-network-operator-chart/crds/sriovnetwork.openshift.io_sriovnetworks.yaml#L9

Reply:
there is this one as well no https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/api/v1/sriovibnetwork_types.go

Reply:
the point is the api group, the one that i sent is not under openshift

Reply:
The api group for the CRDs used by the controller is correct `sriovnetwork.openshift.io`

https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/deployment/sriov-network-operator-chart/crds/sriovnetwork.openshift.io_sriovnetworknodepolicies.yaml#L7

https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/deployment/sriov-network-operator-chart/crds/sriovnetwork.openshift.io_ovsnetworks.yaml#L7

Reply:
oh i see that upstream is using the same api group https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/deployment/sriov-network-operator-chart/crds/sriovnetwork.openshift.io_ovsnetworks.yaml#L9